packages/avocode-email-tagsinput/test/e2e/unique.spec.js

```diff
@@ -0,0 +1,57 @@
+import puppeteer from 'puppeteer'
```

Comment:
Delete this file and instead put the specs in new `describe` block where you actually describe that you're testing uniqueness. I don't think we need separate file for this for now.